var/spack/repos/builtin/packages/gpdb/package.py

```diff
@@ -27,6 +27,7 @@ class Gpdb(AutotoolsPackage):
     version('5.23.0',    sha256='b06a797eb941362d5473b84d5def349b5ce12ce87ab116bea7c74ad193738ae9')
 
     depends_on('zstd')
+    depends_on('py-setuptools@:44.99.99')
```

Comment:
I don't really like this, because it should be up to the concretizer to know that this is necessary. But since the concretizer is currently broken, I'll allow it for now.